### PR TITLE
Plastic flaps fix

### PR DESCRIPTION
**Pull request by Codingale**
Fixes cargo techs being able to steal plastic flaps since they're really potted plants! Tested and working, however varediting anchored to 0 didn't work. Other potted plants work just fine with this change however. Could be that I have no idea how to properly varedit though. (Sidenote, first time editing and also I have no idea why it inserted a line at the end and can't seem to prevent it)

### CODE REVIEW

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -217,4 +217,4 @@
 
 /obj/structure/flora/ausbushes/fullgrass/New()
 	..()
-	icon_state = "fullgrass_[rand(1, 3)]"
\ No newline at end of file
+	icon_state = "fullgrass_[rand(1, 3)]"
```
Comment (Codingale) [inline code/game/objects/structures/flora.dm]: Why did you add a newline?! Damn it Git.

Comment (Lorwp) [inline code/game/objects/structures/flora.dm]: @Codingale [This help page](https://help.github.com/articles/dealing-with-line-endings/) may help

Comment (Codingale) [inline code/game/objects/structures/flora.dm]: @Lorwp it won't break anything will it?

Comment (Lorwp) [inline code/game/objects/structures/flora.dm]: Not too sure, i've never had too much of a problem with line breaks myself so I've never tried anything.

Comment (Codingale) [inline code/game/objects/structures/flora.dm]: How does the code look other than that?

Comment (Lorwp) [inline code/game/objects/structures/flora.dm]: Well, i'll need a moment to test the code myself quickly...One moment.

**Comment by Lorwp** [comment]
Testing, and working :+1:
